### PR TITLE
app-admin/genromfs: add 0.5.7, fix SRC_URI

### PR DESCRIPTION
**Pull request by mm1ke**
simple version bump for `app-admin/genromfs`

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-admin/genromfs/genromfs-0.5.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (ConiKost) [inline app-admin/genromfs/genromfs-0.5.7.ebuild:4]: Please 8 ;-)

Comment (mm1ke) [reply app-admin/genromfs/genromfs-0.5.7.ebuild]: ohh, thx for pointing that out, missed that :)

File: app-admin/genromfs/genromfs-0.5.2.ebuild [hunk 2]
```diff
@@ -7,7 +7,8 @@ inherit toolchain-funcs
 
 DESCRIPTION="Create space-efficient, small, read-only romfs filesystems"
 HOMEPAGE="https://romfs.sourceforge.net/"
-SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/chexum/${PN}/archive/refs/tags/${PV}.tar.gz
+	-> ${P}.gh.tar.gz"
```
Comment (thesamesam) [inline app-admin/genromfs/genromfs-0.5.2.ebuild:11]: If you're changing SRC_URI such that the hash/manifest changes, please include a summary of the diff of the tarballs in the commit message, per https://devmanual.gentoo.org/general-concepts/manifest/index.html#updating-manifest-files.

Comment (thesamesam) [reply app-admin/genromfs/genromfs-0.5.2.ebuild:11]: Please resubmit. I've reverted it in d5b13a5eb13eb91f544fe00417fa5d1d32673d95.

**Pull request closed without merging**